src/common/config.cpp

```diff
@@ -27,21 +94,13 @@ YAML::Node& Config::get() {
   return config_;
 }
 
-void Config::log() {
-  YAML::Emitter out;
-  cli::OutputYaml(config_, out);
-  std::string configString = out.c_str();
-
-  std::vector<std::string> results;
-  boost::algorithm::split(results, configString, boost::is_any_of("\n"));
-  for(auto& r : results)
-    LOG(info, "[config] {}", r);
+const std::vector<DeviceId>& Config::getDevices() {
```

Comment:
I remember a funciton called getDevices() that performs complex parsing of the device string. Is this teh same function, and what happened to it? Or if not, why is there a special accessor for devices, but not for other parameters? Does this belong into this class?

Reply:
Added TODO